index.d.ts

```diff
@@ -2,6 +2,33 @@ import {GlobbyOptions} from 'globby';
 import {Options as CpFileOptions} from 'cp-file';
 
 declare namespace cpy {
+	interface SourceFile {
+		/**
+		Path to file.
+		*/
+		readonly path: string,
+
+		/**
+		Resolved path to file.
+		*/
+		readonly resolvedPath: string,
+
+		/**
+		File name.
+		*/
+		readonly name: string,
+
+		/**
+		File name without extension.
```

Comment:
Can you include an example in each of these doc comments on what it looks like. For example, here it would be:

```
'foo'
```